Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -616,6 +616,67 @@ extension SPNoteEditorViewController {
     }
 }
 
+// MARK: - Search Map
+//
+extension SPNoteEditorViewController {
+
+    /// Show search map keyword ranges
+    ///
+    @objc
+    func showSearchMap(with searchRangeValues: [NSValue]) {
+        createSearchMapViewIfNeeded()
+        // trigger layout to get correct height
+        searchMapView.superview?.layoutSubviews()
```

Comment:
perhaps `view.setNeedsLayout()` instead? (Plus: would it be possible to add a note indicating why is this one needed?)

Reply:
I'm triggering the layout to get correct height of the search map. It is used as a minimum editor height in search terms positions calculation.
I thought to trigger layout and use the height is easier than calculating all margins. That's also why `setNeedsLayout` not gonna work, because the height is used right in that method.

Reply:
Actually got rid of it and now using `editingRect` from the UITextView extension